cmd/server/assets/realm.html

```diff
@@ -356,10 +356,7 @@ <h1>Realm settings</h1>
           Abuse prevention
         </div>
         <div class="card-body">
-          <div class="alert alert-warning">
-            <span class="oi oi-beaker"></span> This feature is still under
-            active development.
-          </div>
+          {{template "beta-notice" .}}
 
           <p>
             Abuse prevention uses the historical record of your realm's past
```

Comment:
I think it might be better if this say why in addition to what.

Reply:
Added